### PR TITLE
feat(server): PR events support specify target branch

### PR DESCRIPTION
**Pull request by zhujian7**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @your-reviewer

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhujian7** [comment]
@supereagle PTAL

File: pkg/server/handler/v1alpha1/webhook.go [hunk 1]
```diff
@@ -136,8 +136,22 @@ func createWorkflowRun(tenant string, wft v1alpha1.WorkflowTrigger, data *scm.Ev
 			}
 		}
 	case scm.PullRequestEventType:
-		if st.PullRequest.Enabled {
+		log.Info("target branch name:", data.Branch)
```
Comment (supereagle) [inline pkg/server/handler/v1alpha1/webhook.go]:
```suggestion
		log.Infof("target branch name: %s", data.Branch)
```

File: pkg/server/handler/v1alpha1/webhook.go [hunk 2]
```diff
@@ -136,8 +136,22 @@ func createWorkflowRun(tenant string, wft v1alpha1.WorkflowTrigger, data *scm.Ev
...
+		if !st.PullRequest.Enabled {
+			break
+		}
+
+		// Always trigger if Branches not defined
```
Comment (supereagle) [inline pkg/server/handler/v1alpha1/webhook.go]:
```suggestion
		// Always trigger if branches are not specified.
```

**Pull request merged**